source/base/TimeSpan.ooc

```diff
@@ -16,6 +16,9 @@ TimeSpan: cover {
 		this _ticks = DateTime timeToTicks(hour, minute, second, millisecond)
 	}
 	negate: func -> This { This new(-1 * this ticks) }
+	elapsedNanoseconds: func -> Long {
+		1_000_000 * this elapsedMilliseconds()
```

Comment:
Why not `this ticks / DateTime nanosecondsPerTick` ?
Current one will return zero for ticks corresponding to less than 1 ms. (as `elapsedMilliseconds()` will round down to zero).